lldb/source/Core/CMakeLists.txt

```
@@ -17,8 +17,8 @@ if (LLDB_ENABLE_CURSES)
 endif()
 
 if (LLVM_BUILD_TELEMETRY)
-   set(TELEMETRY_SOURCES Telemetry.cpp)
    set(TELEMETRY_DEPS Telemetry)
+   add_definitions(-DLLVM_BUILD_TELEMETRY)
```

```suggestion
   add_definitions(-DLLDB_BUILD_TELEMETRY)
```

... since we're defining the macro in lldb code?